src/webgpu/web_platform/external_texture/video.spec.ts

```diff
@@ -27,47 +27,58 @@ const kVideoInfo = /* prettier-ignore */ makeTable(
                                 ['colorSpace',                       'mimeType'] as const,
                                 [   undefined,                        undefined] as const, {
   // All video names
-  'red-green.webmvp8.webm'    : [    'REC601',         'video/webm; codecs=vp8'],
-  'red-green.theora.ogv'      : [    'REC601',       'video/ogg; codecs=theora'],
-  'red-green.mp4'             : [    'REC601',  'video/mp4; codecs=avc1.4d400c'],
-  'red-green.bt601.vp9.webm'  : [    'REC601',         'video/webm; codecs=vp9'],
-  'red-green.bt709.vp9.webm'  : [    'REC709',         'video/webm; codecs=vp9'],
-  'red-green.bt2020.vp9.webm' : [   'REC2020',         'video/webm; codecs=vp9']
+  'four-colors-vp8-bt601.webm'  : [    'REC601',         'video/webm; codecs=vp8'],
+  'four-colors-theora-bt601.ogv': [    'REC601',       'video/ogg; codecs=theora'],
+  'four-colors-h264-bt601.mp4'  : [    'REC601',  'video/mp4; codecs=avc1.4d400c'],
+  'four-colors-vp9-bt601.webm'  : [    'REC601',         'video/webm; codecs=vp9'],
+  'four-colors-vp9-bt709.webm'  : [    'REC709',         'video/webm; codecs=vp9'],
+  'four-colors-vp9-bt2020.webm' : [   'REC2020',         'video/webm; codecs=vp9']
 } as const);
 type VideoName = keyof typeof kVideoInfo;
 type VideoInfo = valueof<typeof kVideoInfo>;
 
+const kBt601Red = new Uint8Array([248, 36, 0, 255]);
+const kBt601Green = new Uint8Array([64, 252, 0, 255]);
+const kBt601Blue = new Uint8Array([26, 35, 255, 255]);
+const kBt601Yellow = new Uint8Array([254, 253, 0, 255]);
```

Comment:
These values of expectation are calculated from https://chromium-review.googlesource.com/c/chromium/src/+/4232318/3/ui/gfx/color_transform_unittest.cc#142

Reply:
Please provide the comments that how these video files generated, where the expected value comes from.

Reply:
I think the reason Shaobo wanted to document  where the expectations come from is so people (that aren't us) can arrive at the same value on their own, and I don't think that linking to a CL with the values hardcoded accomplishes this. One idea is just adding the process described in my [comment ](https://github.com/gpuweb/cts/pull/2242#issuecomment-1430382811)into a comment in the file.

Reply:
> I think the reason Shaobo wanted to document where the expectations come from is so people (that aren't us) can arrive at the same value on their own, and I don't think that linking to a CL with the values hardcoded accomplishes this. One idea is just adding the process described in my [comment ](https://github.com/gpuweb/cts/pull/2242#issuecomment-1430382811)into a comment in the file.

Sounds good, done.